src/client/components/household-information/FinancialDisclosureSection.jsx

```diff
@@ -45,36 +35,37 @@ class FinancialDisclosureSection extends React.Component {
       content = (<fieldset>
         <legend>Financial Disclosure</legend>
         <div className="input-section">
-          {notRequiredMessage}
-
+          <p>In the next 4 sections you will be asked to provide your income information from the most recent tax year. Your most recent
+          tax filing will have all the information you need in it.</p>
           <p>
-            Disclosure allows VA to accurately determine whether certain Veterans
-            will be charged copays for care and medications, their eligibility for
-            other services and enrollment. Veterans are not required to disclose
-            their financial information; however, VA is not currently enrolling new
-            applicants who decline to provide their financial information unless they
-            have other qualifying eligibility factors (i.e., a former Prisoner of
-            War, in <strong>receipt of a Purple Heart, discharged for a disability incurred or
-            aggravated in the line of duty, receiving VA service-connected disability
-            compensation, receiving VA pension, in receipt of Medicaid benefits, or a
-            recently discharged Combat Veteran (e.g., OEF/OIF/OND), who was
-            discharged within the past 5 years are eligible for enrollment without
-            disclosing their financial information)</strong> but like other Veterans may
-            provide their financial information to establish their eligibility for
-            travel assistance, cost-free medications and/or medical care for services
-            unrelated to military experience.
+            Veterans are not required to disclose their financial information; however,
+            <strong> VA is not currently enrolling new applicants who decline to provide their financial information unless
+            they have other qualifying eligibility factors (QEFs).</strong>
```

Comment:
I rather not use bod because I want that to call out the line saying `VA is not currently enrolling new applicants who decline to provide their financial information unless they have other qualifying eligibility factors.` Because if the user doesn't read anything else I want them to read that sentence and I think too much bolding will water-down the effectiveness of using styling for that purpose. Also, the first instance of `qualifying eligibility factors` will already be in bold, so if you are scanning that won't be and inherent link. I am fine to drop the acronyms.